src/web/components/ClientSideTokenGeneration/CstgDomainsTable.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { Meta, Story } from '@storybook/react';
+
+import { CstgDomainsTable, CstgDomainsTableProps } from './CstgDomainsTable';
+
+export default {
+  title: 'CSTG/CstgDomainsTable',
+  component: CstgDomainsTable,
+} as Meta<typeof CstgDomainsTable>;
+
+const Template: Story<CstgDomainsTableProps> = (args) => <CstgDomainsTable {...args} />;
+
+export const WithDomains = Template.bind({});
+
+WithDomains.args = {
+  domains: ['test.com', 'abc.com', '123.com'],
+  onUpdateDomains: (domains) => Promise.resolve(console.log('update domains to:', domains)),
```

Comment:
note that this is a bit confusing when deleting 'test.com', since it will say `update domains to: test.com`. Though I'm not sure of the best way to address that given we don't have separate separate add and delete methods.

Reply:
Ah.... I think I've tested this one... updated